components/elements/inputs/PasswordInput/PasswordInput.test.ts

```diff
@@ -0,0 +1,36 @@
+import { mount, enableAutoUnmount } from '@vue/test-utils'
+import { getByTestId, getHTMLElement } from 'root/testUtils'
+import PasswordInput from './PasswordInput.vue'
+
+function getPasswordInputWrapper() {
+  return mount(PasswordInput)
+}
+
+afterEach(() => {
+  vi.restoreAllMocks()
+})
+
+enableAutoUnmount(afterEach)
+
+describe('<PasswordInput />', () => {
+  it('should render without crashing', () => {
+    const wrapper = getPasswordInputWrapper()
+
+    expect(wrapper.isVisible()).toBe(true)
+  })
+
+  describe('when the hide/show button is clicked', () => {
+    it('changes the password input type to be text', async () => {
+      const wrapper = getPasswordInputWrapper()
+      const passwordInputElement = getHTMLElement(
+        wrapper.get('input'),
+      ) as HTMLInputElement
```

Comment:
side note, man its so annoying working with typing of Elements in TS lol

Reply:
Yeah, sometimes 😅